json-rpc/spec.json

```diff
@@ -1464,6 +1464,11 @@
               "description": "Block hash of the RLP encoding of an uncle block",
               "$ref": "#/components/schemas/Keccak"
             }
+          },
+          "baseFee": {
+            "title": "baseFee",
```

Comment:
```suggestion
            "title": "baseFeePerGas",
```

Reply:
Payment is required.donations to autodeposit email:shaynemichaelhall@gmail.com